docker/ubuntu_18.04-dev.dockerfile

```diff
@@ -145,6 +145,10 @@ RUN if [ "$build_dagmc" = "YES" ]; then \
         && rm -rf DAGMC; \
     fi
 
+# DAGMC path variable
+ENV LD_LIBRARY_PATH $HOME/opt/dagmc/lib:$LD_LIBRARY_PATH
```

Comment:
Is this necessary?

Reply:
@gonuke, It's not necessary. I will revert this change.